src/components/SwapPage.tsx

```diff
@@ -125,35 +125,32 @@ const SwapPage = (props: Props): ReactElement => {
               </a>
             </div>
           </div>
-          <div className="row">
-            <SwapInput
-              tokens={tokenOptions.from.filter(
-                ({ symbol }) => symbol !== toState.symbol,
-              )}
-              onSelect={onChangeFromToken}
-              onChangeAmount={onChangeFromAmount}
-              selected={fromState.symbol}
-              inputValue={fromState.value}
-              inputValueUSD={fromState.valueUSD}
-              isSwapFrom={true}
-            />
-          </div>
+
+          <SwapTokenInput
+            tokens={tokenOptions.from.filter(
+              ({ symbol }) => symbol !== fromState.symbol,
+            )}
+            selected={fromState.symbol}
+            inputValue={fromState.value}
+            inputValueUSD={fromState.valueUSD}
+            onSelect={onChangeFromToken}
+            onChangeAmount={onChangeFromAmount}
+          />
```

Comment:
Don't forget `isSwapFrom={true}` here. This prop could probably have a better name, if one comes to you.